src/modules/outliers/index.js

```diff
@@ -59,6 +59,17 @@ const getExtremeLines = (percentage, xyStats) => {
     return lines
 }
 
+const getMinMaxValue = (outlierHelper, prop, sortFn) => [
```

Comment:
might be cleaner to pass `x` or `y` plus `min` or `max` and let the fn use the correct sort function internally to simplify the use. But as is works too 👍